packages/vuetify/src/components/VCarousel/__tests__/__snapshots__/VCarouselItem.spec.ts.snap

```diff
@@ -1,15 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`VCarouselItem.ts should render component and match snapshot 1`] = `
-<div class="v-window-item"
-     style="display: none;"
->
-</div>
-`;
+exports[`VCarouselItem.ts should render component and match snapshot 1`] = `undefined`;
 
-exports[`VCarouselItem.ts should throw warning when not used inside v-carousel 1`] = `
-<div class="v-window-item"
-     style="display: none;"
->
-</div>
-`;
+exports[`VCarouselItem.ts should throw warning when not used inside v-carousel 1`] = `undefined`;
```

Comment:
doesn't look correct. If you check for `undefined`, use a matcher for that

Reply:
snapshots can all go to hell